test/test_common/environment.h

```diff
@@ -93,6 +93,18 @@ class TestEnvironment {
    */
   static const std::string& nullDevicePath();
 
+  /**
+   * Obtain platform specific new line character(s)
+   * @return absl::string_view platform specific new line character(s)
+   */
+  static constexpr absl::string_view newLine() {
```

Comment:
this can be just a constant, not a function.

Reply:
you mean like this? https://godbolt.org/z/Kdxvo8

Reply:
yes